### PR TITLE
[skip-ci] fix a few more linting errors

### PR DESCRIPTION
**Pull request by bgruening**
FOR CONTRIBUTOR:
* [ ] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [ ] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

**Review by bernt-matthias — COMMENTED** [review]
Thanks a lot. Can you remove the corresponding linter rule from the tool's `.lint_skip` files.

**Comment by bgruening** [comment]
Ah, a rebase was missing. I didn't had many of the skip_lints in my branch.

File: tools/abricate/abricate_list.xml [hunk 1]
```diff
@@ -5,33 +5,28 @@
...
+    <expand macro="edam_ontology"/>
+    <expand macro="xrefs"/>
+    <expand macro="requirements"/>
+    <expand macro="version_command"/>
     <command detect_errors="exit_code"><![CDATA[
         abricate --list > '$report'
     ]]></command>
-
     <inputs>
+        <param name="hidden_input" type="hidden" label="This is to please our linter in this expeptional case"/>
```
Comment (bernt-matthias) [inline tools/abricate/abricate_list.xml:16]: I would prefer to keep the linter exception here.

File: tools/crossmap/.lint_skip [hunk 2]
```diff
@@ -1 +0,0 @@
-TestsExpectNumOutputs
```
Comment (bernt-matthias) [inline tools/crossmap/.lint_skip:1]: Seems that there was no TestsExpectNumOutputs error? Or did you miss to fix it?

**Review by bernt-matthias — COMMENTED** [review]
Looks good. All the XMLOrder fixes should be fine without deployment. Maybe we should pull out the other changes to another PR?

File: tools/crossmap/.lint_skip (hunk 2 above)
Comment (bgruening) [reply tools/crossmap/.lint_skip:1]: I think I updated crossmap in the last week and fixed some stuff ...

**Comment by bgruening** [comment]
@bernt-matthias done!

**Review by bernt-matthias — APPROVED** [review]
Thanks. So I merge with `[skip-ci]`?

**Comment by mvdbeek** [comment]
Attention: deployment cancelled!

https://github.com/galaxyproject/tools-iuc/actions/runs/10834774784

**Pull request merged**